full-stack-tests/presentation/src/learning-snippets/Content.test.ts

```diff
@@ -11,582 +11,1002 @@ import { getFieldByLabel, tryGetFieldByLabel } from "../Utils";
+          keys: new KeySet(),
+          descriptor: {},
+        });
+
+        expect(content!.descriptor.categories).containSubset([{ label: "Axis rotations" }]);
+        expect(content!.descriptor.fields).to.containSubset([{
+          label: "Pitch",
+          category: { label: "Axis rotations" },
         }, {
-          ruleType: RuleTypes.ContentModifier,
-          class: { schemaName: "BisCore", className: "GeometricElement3d" },
-          propertyOverrides: [{
-            // force hide the UserLabel property
-            name: "UserLabel",
-            isDisplayed: false,
-          }, {
-            // force show the Parent property which is hidden by default through ECSchema
-            name: "Parent",
-            isDisplayed: true,
-            doNotHideOtherPropertiesOnDisplayOverride: true,
-          }, {
-            // override label of CodeValue property
-            name: "CodeValue",
-            labelOverride: "Overriden Label",
+          label: "Yaw",
+          category: { label: "Axis rotations" },
+        }, {
+          label: "Roll",
+          category: { label: "Axis rotations" },
+        }]);
+      });
+
+      it("uses `propertyOverrides` attribute", async () => {
+        // __PUBLISH_EXTRACT_START__ ContentInstancesOfSpecificClasses.propertyOverrides.Ruleset
+        // The specification returns content for `bis.SpatialViewDefinition` with one
+        // overriden property label.
+        const ruleset: Ruleset = {
+          id: "example",
+          rules: [{
+            ruleType: RuleTypes.Content,
+            specifications: [{
+              specType: ContentSpecificationTypes.ContentInstancesOfSpecificClasses,
+              classes: { schemaName: "BisCore", classNames: ["ViewDefinition"], arePolymorphic: true },
+              propertyOverrides: [{ name: "Model", labelOverride: "Container Model" }],
+            }],
+          }],
+        };
+        // __PUBLISH_EXTRACT_END__
+
+        // Ensure that the returned content has an overriden property label `Container Model`.
+        const content = await Presentation.presentation.getContent({
+          imodel,
+          rulesetOrId: ruleset,
+          keys: new KeySet(),
+          descriptor: {},
+        });
+
+        expect(content!.descriptor.fields).to.containSubset([
+          { label: "Category Selector" },
+          { label: "Code" },
+          { label: "Container Model" },
+          { label: "Description" },
+          { label: "Display Style" },
+          { label: "Is Private" },
+          { label: "User Label" },
+        ]).and.to.have.lengthOf(7);
+
+        expect(content!.contentSet.length).to.eq(4);
+      });
+
+      it("uses `relatedInstances` attribute", async () => {
+        // __PUBLISH_EXTRACT_START__ ContentInstancesOfSpecificClasses.relatedInstances.Ruleset
+        // The specification returns content for `bis.ModelSelector` and related
+        // `bis.SpatialViewDefinition` instances.
```

Comment:
This is not exactly true. Suggested change:
```suggestion
        // The specification returns content for `bis.ModelSelector` filtered by related
        // `bis.SpatialViewDefinition` instance `Yaw` property value.
```